testing/fuchsia/run_tests.py

```diff
@@ -77,33 +84,7 @@ def run_test(self) -> CompletedProcess:
     return CompletedProcess(args='', returncode=returncode)
 
 
-def bundled_test_runner_of(target_id: str) -> BundledTestRunner:
-  log_dir = os.environ.get('FLUTTER_LOGS_DIR', '/tmp/log')
-  with open(os.path.join(os.path.dirname(__file__), 'test_suites.yaml'),
-            'r') as file:
-    tests = yaml.safe_load(file)
-  # TODO(zijiehe-google-com): Run tests with extra test arguments,
-  # https://github.com/flutter/flutter/issues/140179.
-  tests = list(
-      filter(
-          lambda test: test['test_command'].startswith('test run ') and test[
-              'test_command'].endswith('.cm'), tests
-      )
-  )
-  # TODO(zijiehe-google-com): Run tests with dart aot,
-  # https://github.com/flutter/flutter/issues/140179.
-  tests = list(
-      filter(
-          lambda test: 'run_with_dart_aot' not in test or test[
-              'run_with_dart_aot'] != 'true', tests
-      )
-  )
-  tests = list(
-      filter(
-          lambda test: 'variant' not in test or VARIANT == test['variant'],
-          tests
-      )
-  )
+def resolve_packages(tests: List[Any]) -> Set[str]:
```

Comment:
Give correct typing rather than `Any`

Reply:
PyYaml returns multiple types from the `safe_load` function, so likely we can only say it returns Iterable[Mapping[str, str|list[str]]] according to the yaml file we are using. But `|` / `union` type is not supported by 3.8 which is used by chromium now which requires 3.10.